### DISCUSSION TITLE
Who can compile Android project with Android Studio?

### DISCUSSION THREAD

**Discussion by RogerYe** [flyinghead/flycast#755] [Q&A]
I compiled it with Android Studio Dolphin|2021.3.1, and got these message:

> Task :flycast:configureCMakeRelWithDebInfo[armeabi-v7a]
C/C++: prefabcom.google.prefab.api.NoMatchingLibraryException: No compatible library found for //oboe/oboe. Rejected the following libraries:
C/C++: prefabandroid.arm64-v8a: User is targeting armeabi-v7a but library is for arm64-v8a
C/C++: prefabandroid.armeabi-v7a: User is using a static STL but library requires a shared STL
C/C++: prefabandroid.x86: User is targeting armeabi-v7a but library is for x86
C/C++: prefabandroid.x86_64: User is targeting armeabi-v7a but library is for x86_64
C/C++: release|armeabi-v7a :CMake Deprecation Warning at CMakeLists.txt:364 (cmake_policy):
C/C++: release|armeabi-v7a :  The OLD behavior for policy CMP0026 will be removed from a future version
C/C++: release|armeabi-v7a :  of CMake.
C/C++: release|armeabi-v7a :  The cmake-policies(7) manual explains that the OLD behaviors of all
C/C++: release|armeabi-v7a :  policies are deprecated and that a policy should be set to OLD only under
C/C++: release|armeabi-v7a :  specific short-term circumstances.  Projects should be ported to the NEW
C/C++: release|armeabi-v7a :  behavior and not rely on setting a policy to OLD.
C/C++: release|armeabi-v7a :CMake Error at core/deps/oboe/gamesdk.cmake:117 (message):
C/C++: release|armeabi-v7a :  Can't find any library in
C/C++: release|armeabi-v7a :  "core/deps/oboe/libs/armeabi-v7a_API28_NDK21_cpp_static_RelWithDebInfo".
C/C++: release|armeabi-v7a :  Are you sure you are using a supported Android SDK/NDK and STL variant?
C/C++: release|armeabi-v7a :Call Stack (most recent call first):
C/C++: release|armeabi-v7a :  CMakeLists.txt:481 (add_gamesdk_target)

> Task :flycast:configureCMakeRelWithDebInfo[armeabi-v7a] FAILED
===================================================================
Many thanks to any help. More respect to Rinnegatamante.

**Comment by flyinghead** [comment]
That's probably because windows doesn't support symbolic links. Copy `armeabi-v7a_API28_NDK21_cpp_static_Release` to `armeabi-v7a_API28_NDK21_cpp_static_RelWithDebInfo` in `core/deps/oboe/libs`.

**Reply by RogerYe** [reply]
thanks a lot
